CHANGELOG.md

```diff
@@ -6,454 +6,494 @@ All notable changes to this project will be documented in this file.
-  - Tenancy: Splitting MiqGroup, Part 2
-    - Filters moved to to Entitlement model
-    - Enabler for sharing entitlements   
-  - MiqExpression Refactoring
-- Providers
-  - Core
-    - Remove provider specific constants
-    - Ask whether provider supports VM architecture instead of assuming support by provider type
-  - Ansible: Automate method updated to pass JobTemplate “Extra Variables” defined in the Provision Task
-  - Hawkular
-      - Upgrade of Hawkular gem to 2.3.0
-      - Skip unreachable middleware providers when reporting
-      - Add re-checking authentication status functionality/button
-      - Refactor infrastructure for easier configuration
-      - Optimization and enhancement of event fetching
-  - vSphere Host storage device inventory collection improvements
-- REST API
-  - Updated /api entrypoint so collection list is sorted
-  - API CLI moved to tools/rest_api.rb
-  - Update API CLI to support the HTTP OPTIONS method.
-- User Interface
-  - Dynatree replaced with bootstrap-treeview
-  - Converted to TreeBuilder - Snapshot, Policy, Policy RSOP, C&U Build Datastores and Clusters/Hosts, Automate Results
-  - CodeMirror version updated (used for text/yaml editors)
-  - Default Filters tree converted to TreeBuilder - more on the way
-  - Cloud Key Pair form converted to AngularJS (Dana - UX team)
-  - Toolbars:Cleaned up partials, YAML -> classes
-  - Provider Forms: Credentials Validation improvements
-  - Updated PatternFly to v3.11.0
-  - Summary Screen styling updates
+### Automate
+
+- Description for Datastore Reset action now includes list of target domains
+- Simulation: Updated defaults
+  - Entry-point: `/System/Process/Request` (Previous value of “Automation”)
+  - Execute Method: Enabled
+- Infrastructure Provision: Updated memory values for VM provisioning dialogs to 1, 2, 4, 8, 12, 16, 32 GB
+- Generic Object: Model refactoring/cleanup, use PostgreSQL jsonb column
+- Changed Automate import to enable system domains
+- Google schema changes in Cloud Provision Method class
+
+### Performance
+
+- Page rendering
+  - Compute -> Infrastructure -> Virtual Machines: 9% faster, 32% fewer rows tested on 3k active vms and 3k archived vms
+  - Services -> My Services: 60% faster, 98% fewer queries, 32% fewer db rows returned
+  - Services -> Workloads -> All VMs page load time reduced from 93,770ms to 524ms (99%) with a test of 20,000 VMs
+- `Ownershipmixin`
+  - Filtering now done in SQL
+  - 99.5% faster (93.8s -> 0.5s) testing
+    - VMs / All VMs / VMs I Own
+    - VMs / All VMs / VMs in My LDAP Group
+- Reduced the time and memory required to schedule Capacity and Utilization data collection.
+- Capacity and Utlization improvements included reduced number of SQL queries and number of objects
+- Improved tag processing for Alert Profiles
+- UI Performance: specific pages targeted resulted in up to 98% reduction in rendering for those pages
+- Do not reload miq server in tree builder
```

Comment:
This seems like a fragment...needs more detail (at least percentages or time or something)  cc @dmetzger57